fe1-web/parts/Wallet.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet, TextInput, TextStyle, View, ViewStyle,
+} from 'react-native';
+import styleContainer from 'styles/stylesheets/container';
+import STRINGS from 'res/strings';
+import TextBlock from 'components/TextBlock';
+import WideButtonView from 'components/WideButtonView';
+import { Spacing, Typography } from 'styles';
+import { HDWallet } from 'model/objects/HDWallet';
+
+const styles = StyleSheet.create({
+  textInput: {
+    ...Typography.base,
+    borderBottomWidth: 2,
+    marginVertical: Spacing.s,
+    marginHorizontal: Spacing.xl,
+  } as TextStyle,
+  viewTop: {
+    justifyContent: 'flex-start',
+  } as ViewStyle,
+  viewBottom: {
+    justifyContent: 'flex-end',
+  } as ViewStyle,
+});
+
+const wallet: HDWallet = new HDWallet();
+
+const Wallet = () => {
+  /* this is the UI state of the wallet, starts null (namely homepage)
+     the states can be: null | show_seed | set_seed | initialized */
+  const [walletState, setWalletState] = useState(STRINGS.wallet_state_null);
+  /* used to set the mnemonic seed inserted by the user */
+  const [seed, setSeed] = useState('');
+  /* boolean set to true if the token recover process is finished */
+  const [tokensRecovered, setTokensRecovered] = useState(false);
+
+  /* all the below are temporarily used for demonstration purposes (showing seed in UI) */
+  const [showHideSeedTitle, setShowHideSeedTitle] = useState(STRINGS.show_seed_title);
```

Comment:
if you're using this just to change a button title, you can just have two buttons, and you display either based on a boolean value.